caddyconfig/caddyfile/dispenser.go

```diff
@@ -415,7 +415,7 @@ func (d *Dispenser) EOFErr() error {
 
 // Err generates a custom parse-time error with a message of msg.
 func (d *Dispenser) Err(msg string) error {
-	return d.Errf(msg)
+	return d.WrapErr(errors.New(msg))
```

Comment:
The `d.Errf` calls `d.WrapErr` (see the method directly below this). Was there an issue prompting the change?

Reply:
It was a (justified IMHO) `govet` error that was making the `golangci-lint` job red:

```
  Error: caddyconfig/caddyfile/dispenser.go:418:16: printf: non-constant format string in call to (*github.com/caddyserver/caddy/v2/caddyconfig/caddyfile.Dispenser).Errf (govet)
  	return d.Errf(msg)
```